orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/utils/OrientDBBackendTckHelper.java

```diff
@@ -6,18 +6,52 @@
  */
 package org.hibernate.datastore.ogm.orientdb.utils;
 
+import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx;
+import org.apache.log4j.Logger;
+import static org.hibernate.datastore.ogm.orientdb.OrientDBSimpleTest.MEMORY_TEST;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.extensions.cpsuite.ClasspathSuite;
 import org.junit.runner.RunWith;
 
 /**
- * Helper class allowing you to run all or any specified subset of test available on the classpath.
- *
- * This method is for example useful to run all or parts of the <i>backendtck</i>.
+ * Helper class allowing you to run all or any specified subset of test available on the classpath. This method is for
+ * example useful to run all or parts of the <i>backendtck</i>.
  *
  * @author Hardy Ferentschik
  */
 @RunWith(ClasspathSuite.class)
-//@ClasspathSuite.ClassnameFilters({ "org.hibernate.ogm.backendtck.*" })
+// @ClasspathSuite.ClassnameFilters({ "org.hibernate.ogm.backendtck.*" })
 @ClasspathSuite.ClassnameFilters({ ".*BuiltInTypeTest" })
 public class OrientDBBackendTckHelper {
+
+	private static final Logger log = Logger.getLogger( OrientDBBackendTckHelper.class.getName() );
+
+	private static ODatabaseDocumentTx db;
+
+	@BeforeClass
+	public static void setUpClass() {
+		log.info( "setUpClass" );
+		db = MemoryDBUtil.createDbFactory( MEMORY_TEST );
```

Comment:
There shouldn';t be any need for this here.

The db is started by the datastore provider when you run the other tests and closed accordingly.
This class should only be a utility to run the existing tests